docs/dev/trs-setup.md

```diff
@@ -1,8 +1,27 @@
 # TRACE Server Setup Guide
 
-## Before you begin
+## Pre-requisites
 
-This guide assumes that you have a system running Ubuntu 24.04 LTS. If you are using a different operating system, you may need to adjust the commands accordingly. We also assume that you can execute commands as the root user. Places where you need to run a command as the root user will be indicated with `sudo`. We recommend that you create a dedicated user for running the TRACE Server. In our example we will use the username `ubuntu`.
+This guide will help you set up a TRACE server and infrastructure. This involves:
+
+- Having a way to sign the TROs (Trace Record Objects) that are generated by the TRACE server. TRACE allows for GPG and X.509. See [Signing](signing) for more information.
```

Comment:
```suggestion
- Having a way to sign the TROs (Trace Record Objects) that are generated by the TRACE server. TRACE allows for GPG and X.509. See [Signing](./signing.md) for more information.
```
Otherwise `jupyter-book` complains about ambiguous reference